src/main/java/com/authlete/jaxrs/BaseResourceEndpoint.java

```diff
@@ -95,14 +95,14 @@ public String extractAccessToken(String authorization, String accessTokenInReque
      */
     public AccessTokenInfo validateAccessToken(AuthleteApi api, String accessToken) throws WebApplicationException
     {
-        return validateAccessToken(api, accessToken, null, null);
+        return validateAccessToken(api, accessToken, null, null, null);
     }
 
 
     /**
      * Validate an access token. This method is an alias of {@link
      * #validateAccessToken(AuthleteApi, String, String[], String)
```

Comment:
`#validateAccessToken(AuthleteApi, String, String[], String)` to
`#validateAccessToken(AuthleteApi, String, String[], String, String)`